content/podcast/2023/ep14_summer_podcast_release_highlights.md

```diff
@@ -0,0 +1,49 @@
+title=Summer podcast season, Nextflow release highlights.
+episode=14
+description=Welcome back to the Summer 2023 series of the Channels Podcast!  In this episode we kick things off with some of the highlights from the latest Nextflow releases. There were a lot of goodies in there..
+date=2023-05-22
+type=podcast
+subtype=News and Views
+youtubeid=oxZblH02MCA
+image=img/podcast_ep14.jpg
```

Comment:
Just realised that you didn't upload this share image 😉 

No worries, will push it in a PR I'm doing for the next episode now. But just FYI for next time..